localization/english/event_localization/wc_arthas_expedition_events_l_english.yml

```diff
@@ -0,0 +1,136 @@
+﻿l_english:
+ activity_arthas_expedition:0 "Outbreak Investigation"
+ activity_arthas_expedition_name:0 "[ACTIVITY.GetOwner.GetShortUINamePossessive|U] Outbreak Expedition"
+ activity_arthas_expedition_desc:0 "An expedition sent to investigate the plague outbreak and who is behind it."
+ activity_arthas_expedition_owner:0 "Leader"
+ activity_arthas_expedition_invalid:0 "[activity.GetName] Invalid"
+ activity_arthas_expedition_invalid_tooltip:0 "#V $activity_arthas_expedition$#! becomes invalid"
+
+ plague_outcome.5001.terenas_beginning_alliance:0 "For decades, I have ruled over the Alliance of Lordaeron. Disturbances have pushed our alliance to the brink of collapse. Nevertheless, by my wisdom we persevere.\n\nAmbassadors from all kingdoms stand before me. The orcs are rallying, but for what ends I cannot tell."
+ plague_outcome.5001.terenas_beginning_common:0 "For decades, I have ruled these lands. Disturbances have pushed our [lordaeron.GetTierAsName] to the brink of collapse. Nevertheless, by my wisdom we persevere.\n\nAmbassadors stand before my council."
+ plague_outcome.5001.terenas_ending:0 "It is time to act. What now? A raven morphs into a hooded man in front of me, declaring that the world is poised upon the brink of war.\n\nI shout in response: "Our lands are beset by conflict, but it shall be we who decide how best to protect our people, not you! Now begone!""
+ wc_arthas_story_9_1_OPT_A:0 "[arthas.GetFirstName|U], my [ROOT.Char.Custom2('RelationToMeShort', arthas)], find out what is going on."
+ wc_arthas_story_9_1_OPT_B:0 "[arthas.GetFirstName|U] don't dare to go there!"
+
+ wc_arthas_story_10_TITLE:0 "Outbreak Expedition"
+ wc_arthas_story_10_DESC_beggining:0 "[terenas.GetTitledFirstName], my [ROOT.Char.Custom2('RelationToMeShort', terenas.)], calls upon me to investigate the rumours of a plague in the northlands. My entire upbringing has prepared me for this moment. It is time to fulfill my duty and protect those in need of help, whatever it takes."
```

Comment:
rumors is done in [8024120](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/commits/80241204400e332c8b76ad67c6db16ce5ffccfec), event name left like this because it's the same in the code.